src/index.ts

```diff
@@ -160,7 +160,72 @@ const transform = async ({
     exit(1);
   }
 
-  const functionsDir = resolve(".vercel/output/functions");
+  // RoutesManifest.version and RoutesManifest.basePath are the only fields accessed
```

Comment:
I am not a big fan of including this if it's not really needed, could you instead just set the version and basepath?

also the `3` is used here and also below, I think that setting up a constant and using that instead would make the code more robust and clear.

Basically I am thinking of something along these lines:
![Screenshot 2023-03-10 at 21 13 55](https://user-images.githubusercontent.com/61631103/224429826-cfbf867e-66b4-4754-9a85-b5ea5cdf0f58.png)

What do you think? 🙂 

(PS: I am not sure if I got the right url there but you get the gist 🙂)